src/Compilers/Core/Portable/CodeAnalysis.csproj

```diff
@@ -29,6 +29,7 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <Visible>false</Visible>
     </Content>
+    <Compile Include="CommandLine\CommitHashAttribute.cs" />
```

Comment:
>CommandLine\ [](start = 22, length = 12)

I am not sure if the file should be in CommandLine folder. We are not adding it to csc/vbc.